src/Stack/Build/Execute.hs

```diff
@@ -1850,7 +1850,7 @@ singleTest topts testsToRun ac ee task installedMap = do
                         "package-db " <> fromString localDBPath <> "\n" <>
                         foldMap (\ghcId -> "package-id " <> RIO.display (unGhcPkgId ghcId) <> "\n")
                             (thGhcId:packageIds)
-                logInfo $  "package ids: " <> RIO.displayShow packageIds
+                logDebug $  "package ids: " <> RIO.displayShow packageIds
```

Comment:
```suggestion
```
I should have removed it actually before committing

Reply:
Fixed